ui/page/dcrdex/dex_onboarding_page.go

```diff
@@ -201,7 +205,7 @@ func NewDEXOnboarding(l *load.Load, existingDEXServer string) *DEXOnboarding {
 
 	pg.isLoading = true
 	go func() {
-		<-pg.dexc.Ready()
+		<-dexc.Ready()
 		pg.isLoading = false
```

Comment:
This goroutine should be in OnNavigatedTo(). That the page was intialized doesn't mean it will definitely be displayed.

Also, the display should be refreshed after toggling pg.isLoading.